HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,711 @@
+
+At any time after they've received GameNFT token value, Alice, Bob, Carol or Dave could change their mind and reject the token back to the GameNFT token treasury account.
+
+> A wallet may or may not provide the functionality described in this HIP. It is outside the scope of the HIP to define how wallets will handle pending transfers. The described scenario is one possible workflow.
+
+
+## User stories
+
+### Crypto Transfer User Stories
+
+#### Sender
+
+1. As an hts-token-sender, I want to send the token using the CryptoTransfer transaction to a receiver who has associated with this token.
+2. As an hts-token-sender, I want to send the token using the CryptoTransfer transaction to a receiver who has not associated with this token but has an open auto-association slot.
+3. As an hts-token-sender, I understand my call will fail if I attempt to send an HTS token using the CryptoTransfer transaction to a receiver who has not associated  with this token and has no open auto-association slot.
+4. As an hts-token-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of a transfer to an account that is already associated with a token to reduce the friction by a receiving account
+5. As an hts-token-sender I want to cover the fees (association & rent until the end of the first full auto renewal period) of a transfer that consumes an automatic token slot to reduce the friction by a receiving account
+
+#### Receiver
+
+1. As a receiver with no open auto association slots, I don’t want to receive any tokens from a sender using CryptoTransfer to transfer tokens which I have not previously associated with.
+2. As a receiver with open auto association slots, I want to automatically receive tokens from a sender using CryptoTransfers to transfer tokens I have not previously associated with. 
+3. As a user with receiver signature required enabled any airdrops sent to me will fail regardless of open slots.
+
+### Token Airdrop User Stories
+
+#### Sender
+
+1. As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has already associated with the token. I expect this should work exactly like the crypto transfer.
+2. As an hts-token-airdrop-sender,  I want to airdrop a token using to a receiver who has not associated with the token but has an open autoAssociation slot. I expect this should work exactly like the cryptoTransfer.
+3. As an hts-token-airdrop-sender,  I want to airdrop a token to a receiver who has not associated with the token and has no open autoAssociation slots. I expect this transfer will be marked as pending that the receiver can claim at any time if it is still available at that time.
```

Comment:
As an hts-token-airdrop-sender, I want to airdrop a token to a receiver who has not associated with the token and has no open autoAssociation slots. I expect this transfer will be marked as pending `so` that the receiver can claim at any time if it is still available at that time.